Doc/tutorial/classes.rst

```diff
@@ -638,7 +638,7 @@ multiple base classes looks like this::
        <statement-N>
 
 For most purposes, in the simplest cases, you can think of the search for
-attributes inherited from a parent class as depth-first, left-to-right, not
+attributes inherited from a parent class as breadth-first, left-to-right, not
```

Comment:
What about the following example in L643-L645?

Reply:
> What about the following example in L643-L645?

Good point.  Have been thinking about this more.  Maybe should cancel the PR and discuss this portion of the docs further in the issue ticket #125904?